2.3.0 Changes

cs-abhishek-shukla opened the pull request:
**2.3.0 Changes**
- Updated release notes, included information related to the 3 mantis IDs.
- Updated upgrade instructions on configuring **NIST** and running the wizard again
- Removed _Fortinet Fortiguard Outbreak_ connector **configuration info** from `setup.md`
- Update version information on `README.md`

cs-sujeet-ahirrao commented on docs/setup.md:
```diff
@@ -67,11 +69,11 @@ Click the button **Setup Outbreak Response Framework** on the Outbreak Response
 
-The hunt activities require searching for adversaries and their tactics, within an environment, against existing information available in the Threat Intel Platform's database (TIP). The Threat Detection Integration sources help run the threat hunt activities and are an important part of the *Outbreak Response Framework*.
+The hunt activities require searching for adversaries and their tactics, within an environment, against existing information available in the Threat Intel Platform's database (TIP). The *Threat Hunting Integrations* help run the threat hunt activities and are an important part of the *Outbreak Response Framework*.
 
```
in setup.md - **Configure the Fortinet FortiAnalyzer**: Refer to [NIST NVD] should be - **Configure the Fortinet FortiAnalyzer**: Refer to [FAZ]
<img width="1036" height="421" alt="image" src="https://github.com/user-attachments/assets/65156802-cadd-49d7-b6cc-3fbc43e5767f" />

cs-abhishek-shukla replied:
Fixed. Thanks

cs-sujeet-ahirrao commented on docs/upgrade-instructions.md:
```diff
@@ -15,7 +15,9 @@ This section points out some actions to take after the upgrade to **Outbreak Res
 ## After Upgrade
```
Please review upgrade-instructions with Rahul

cs-sujeet-ahirrao commented on release_notes.md:
```diff
@@ -1,103 +1,25 @@
->[!NOTE]
->This solution pack requires FortiSOAR `v7.6.1` and later.
+> [!NOTE]
+> This solution pack requires FortiSOAR `v7.6.4` and later.
```
<p>Min Compatability ⁠<readonly class="skipProofing" contenteditable="false" spellcheck="false" itemtype="http://schema.skype.com/Mention"><mention aria-label="Abhishek" type="person" mri="8:orgid:ac68388c-82c1-427b-903d-d45dea14a879">Abhishek</mention></readonly>&nbsp;<readonly class="skipProofing" contenteditable="false" spellcheck="false" itemtype="http://schema.skype.com/Mention"><mention aria-label="Shukla" type="person" mri="8:orgid:ac68388c-82c1-427b-903d-d45dea14a879">Shukla</mention></readonly></p><figure class="table">
SOAR Framework | v3.3.0
-- | --
Threat Intel Management | v3.0.0
Vulnerability Management | v2.2.0
FortiSOAR | v7.6.4

</figure><p>&nbsp;</p>

cs-abhishek-shukla replied:
Changed
In `setup.md`
| Name                     | Type          | Version           | Purpose                                |
|:-------------------------|:--------------|:------------------|:---------------------------------------|
| SOAR Framework           | Solution Pack | v3.3.0 and later  | Required for Incident Response modules |
| Threat Intel Management  | Solution Pack | v3.0.0  and later | Required to ingest threat feeds        |
| Vulnerability Management | Solution Pack | v2.2.0  and later | Required to ingest CVEs for KEVs       |

In `release_notes.md`
> [!NOTE]
> This solution pack requires FortiSOAR `v7.6.4` and later.

In `README.md`
- **Version**: 2.2.1![](./docs/res/icon-preview.svg)
- **Certified**: Yes
- **Publisher**: Fortinet
- **Compatible Version**: 7.6.4 and later
- [Release Notes](./release_notes.md)

cs-rahul-vishwakarma commented on docs/setup.md:
```diff
@@ -14,34 +14,36 @@ The **Outbreak Response Framework** solution pack depends on the following solut
 > [!Important]
-> After an upgrade, you must run the **Outbreak Response Framework** configuration wizard again.
+> After an upgrade:
+> 1. Configure the NIST NVD connector.
```
"1. Configure the NIST NVD connector."

Above can be removed. NIST NVD configuration is a part of Outbreak Configutation Wizard.

cs-abhishek-shukla replied:
Removed

cs-rahul-vishwakarma commented on release_notes.md:
```diff
@@ -1,103 +1,29 @@
+- **FortiTIP Dashboards Integrated**: These dashboards contain outbreak-specific data, making them more relevant when included directly in the **Outbreak Response Framework**. Analysts can now have a consolidated view of outbreak intelligence without needing to switch contexts.
 
-### Wizard Enhancements
+  - The **FortiTIP Overview** dashboard has been moved into *Outbreak Response Framework* and renamed to **Threat Intel Overview**.
```
The dashboards `Threat Intel Overview` and `Threat Intel Insights Report` are newly added dashboards in this version.

- `Threat Intel Overview`: This dashboard presents a focused view of critical threat intelligence and security posture data.
- `Threat Intel Insights Report`: The dashboard provides a comprehensive overview of threat intelligence and security activity, segmented into key focus areas.

cs-rahul-vishwakarma replied:
More details about these dashboards are available in FortiTIP Doc: 
https://docs.fortinet.com/document/fortitip-cloud/25.2.0/administration-guide/124077/dashboards

FortiTIP Overview = Threat Intel Overview
FortiTIP In-Depth Report = Threat Intel Insights Report

cs-rahul-vishwakarma replied:
Information about the same should be added in the `background-information.md` file as well, in the section - **Outbreak Dashboard**
<img width="1920" height="1080" alt="dashboard-threat-intel-overview" src="https://github.com/user-attachments/assets/5a458731-03d5-4073-a329-3878bae0489e" />
<img width="1930" height="2325" alt="dashboard-threat-intel-insights-report" src="https://github.com/user-attachments/assets/844ea4e2-a970-4762-a058-47831857d318" />

cs-abhishek-shukla replied:
"Threat Intel". Should be part of TIM, not ORF

cs-rahul-vishwakarma replied:
Those dashboards combine data from both ORF as well as TIM.
 
Since ORF also installs TIM SP, these dashboards are included in ORF, and not the other way around.

cs-rahul-vishwakarma commented on release_notes.md:
```diff
@@ -1,103 +1,29 @@
+  - The **Get Outbreak Alert Threat Actors and Link Threat Reports** playbook now establishes a many-to-many relationship with the **Threat Actor** module ensuring outbreak alerts are enriched with deeper adversary context, enabling analysts to quickly see which actors are associated with which reports and alerts.
 
-- The wizard now supports configuring the connector on an agent.
+### Fixes
```
Add following fixes:
- Fixed an issue where Outbreak Installation playbooks were failing intermittently.

cs-abhishek-shukla replied:
Added

cs-abhishek-shukla committed:
2.3.0 changes
    - Updated release notes, included information related to the 3 mantis IDs.
    - Updated upgrade instructions on configuring NIST and running the wizard again
    - Removed Fortinet Fortiguard Outbreak connector configuration info from setup.md
    - Update version information on README.md
    - Review comments addressed
    - Added new dashboards' information
